kernel/mount.c

```diff
@@ -25,9 +27,11 @@
 #include <myst/verity.h>
 
 #define MOUNT_TABLE_SIZE 8
+#define AUTOMOUNT_DIR "/tmp/automounts"
```

Comment:
Maybe move this to ``/run/mystikos/automounts``.

Reply:
Done.